### ISSUE TITLE
Menu Missing on Responsive Narrow Layout

### ISSUE THREAD

**Issue by technicaltitch** [mrin9/RapiDoc#595]
When viewing in a narrow browser window, there is no longer any menu for navigating between endpoints (unless I'm mistaken?). This is especially problematic for the focused render style, as there is no way to move between endpoints, as far as I can tell.

I am using `<rapi-doc server-url="https://fdw.fews.net/"  use-path-in-nav-bar="true">`, although I've tried various variations.

An auto-complete search bar fixed to the top of the window, styled like the header bar, that allows you to type a few characters and jump to an endpoint, would be out of this world incredible. A hamburger icon would presumably be much simpler and perfectly adequate.

**Comment by prashantevolvus** [comment]
Yes if you view the below link in iPhone, you will not be able to browse to the API as there is no Menu 

 render-style="focused"

https://mrin9.github.io/RapiDoc/examples/focused-mode.html

**Comment by mrin9** [comment]
I am looking for an PR on this one, else this will be in our queue for some time

**Comment by martinmckenna** [comment]
this is kinda crazy the in the year 2024, mobile-friendly UI updates haven't been added

**Comment by martinmckenna** [comment]
> > this is kinda crazy the in the year 2024, mobile-friendly UI updates haven't been added. came here looking for an answer on this, but it's easier just to migrate away from rapidoc and use something else
> 
> 
> 
> @martinmckenna that's actually why we've built out a fork of this project with many quality of life improvements over at the https://github.com/authress-engineering/openapi-explorer

sick i'll take a peek!
